development/browser_test/composer.json

```diff
@@ -1,11 +1,13 @@
 {
   "minimum-stability": "stable",
   "require": {
-    "php": ">=7.4.0"
+    "php": ">=8.1.0"
   },
   "require-dev": {
     "roave/security-advisories": "dev-master",
-    "codeception/codeception": "^3.1.3",
+    "codeception/codeception": "^5.0.2",
+    "codeception/module-webdriver": "dev-master",
+    "codeception/module-asserts": "dev-master",
```

Comment:
Using dev-master tends to make builds more fragile, since it would accept backwards-breaking changes. It may be better to use...
```suggestion
    "codeception/module-webdriver": "^3.1",
    "codeception/module-asserts": "^3.0",
```
... so that we get what is current right now, accepting any non-breaking updates in the future.

If there was a specified reason to use `dev-master`, feel free to say so, as I'm not as familiar with the current work in this repo.